NorthstarDLL/mods/modmanager.cpp

```diff
@@ -703,11 +706,21 @@ void ModManager::LoadMods()
 	// sort by load prio, lowest-highest
 	std::sort(m_LoadedMods.begin(), m_LoadedMods.end(), [](Mod& a, Mod& b) { return a.LoadPriority < b.LoadPriority; });
 
+	// This is used to check if some mods have a folder but no entry in enabledmods.json
+	bool newModsDetected = false;
+
 	for (Mod& mod : m_LoadedMods)
 	{
 		if (!mod.m_bEnabled)
 			continue;
 
+		// Add mod entry to enabledmods.json if it doesn't exist
+		if (!mod.m_bIsRemote && !m_EnabledModsCfg.HasMember(mod.Name.c_str()))
```

Comment:
I assume that means we ignore remote mods when adding to `enabledmods.json`? ^^

Reply:
Exactly yeah, as discussed in https://github.com/R2Northstar/NorthstarLauncher/pull/410#issuecomment-1426284180